gitlab/assets/configuration/spec.yaml

```
@@ -24,6 +24,7 @@ files:
         send_distribution_counts_as_monotonic.value.example: true
         send_monotonic_counter.enabled: true
         send_monotonic_counter.value.example: true
```

```suggestion
        send_monotonic_counter.value.example: true
        send_monotonic_counter.value.default: true
```
Nit- can we keep the default value as true in case the models are used in the future?

---

We leave the send_monotonic_counter.value.default to true like always, the change is to add the value.display_default: false , this automatically updates the defaults.py file , so I don't think we can update the one without the other.

---

I've tested setting both and this will set the "default" in the example `conf.yaml` to False but not `defaults.py`

---

But when running the ddev validate models the defaults.py get changed. is it okay to skip this command then?

---

Update: by setting send_monotonic_counter.value.default: true and send_monotonic_counter.value.dispaly_default: false, there is no need to update the default.py, the default value from the models will be true, but it will be set to false by the check code itself (therefore the displayed default value : false reflects the current behaviour which is false).